ganttproject/src/biz/ganttproject/impex/csv/SpreadsheetWriter.java

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2017 Alexandr Kurutin, BarD Software s.r.o
+
+This file is part of GanttProject, an opensource project management tool.
+
+GanttProject is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+GanttProject is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package biz.ganttproject.impex.csv;
+
+import java.io.Closeable;
+import java.io.Flushable;
+import java.io.IOException;
+
+/**
+ * @author akurutin on 04.04.2017.
+ */
+public interface SpreadsheetWriter extends Flushable, Closeable {
+  void print(String value) throws IOException;
+
+  void println() throws IOException;
+
+  void close() throws IOException;
```

Comment:
why redurant? I can rename the Flush to save, since the old code used a flush() and I did not change it. XlsWriter use flush() it like `myWorkbook.write(myStream);` and CsvWriter like `myCsvPrinter.flush();`